terraform-module/modules/frontend-spa-cdn/vars.tf

```
@@ -58,3 +58,9 @@ variable "is_robots_indexing_allowed" {
   default     = true
   type        = bool
 }
+
+variable "cloudwatch_access_logs" {
+  description = "Enable CloudWatch access logs"
+  default     = false
```

I would do `true` by default, so by default you have logs. but you can always opt-out.
```suggestion
  default     = true
```